Doc/whatsnew/3.7.rst

```diff
@@ -273,6 +273,10 @@ Changes in the Python API
   ``makedirs()``.
   (Contributed by Serhiy Storchaka in :issue:`19930`.)
 
+* Now wildcard is supported in hostname when it is one and only character in the
+  left most segment of hostname in second argument of :meth:`match_hostname`.
```

Comment:
I'd change :meth:\`match_hostname\` to :meth:\`ssl.match_hostname\`. This would create a link to the `ssl.match_hostname()` documentation.